front/logger/logger.ts

```diff
@@ -52,9 +54,10 @@ export default logger;
 export type { Logger } from "pino";
 
 export function auditLog(
+  author: UserResource | null,
   data: Record<string, unknown>,
   message: string,
   auditLogger = logger
 ) {
```

Comment:
```suggestion
export function auditLog(
data: { author: UserType | "no-author" } & Record<string, unknown> ,
message: string,
auditLogger = logger
) {
```
using a type vs a resource, a resource implies we're going to operate while a type is immutable which reduces reader anguish :)

using an explicit "no-author" if we want to be extra sure people don't put undefined or null without thinking;